cli/command/deploy.go

```diff
@@ -302,6 +302,7 @@ func displayDeployTitle(curveadm *cli.CurveAdm, dcs []*topology.DeployConfig) {
 	curveadm.WriteOutln("Cluster Name    : %s", curveadm.ClusterName())
 	curveadm.WriteOutln("Cluster Kind    : %s", dcs[0].GetKind())
 	curveadm.WriteOutln("Cluster Services: %s", serviceStats(dcs))
+	curveadm.WriteOutln("Cluster Type    : %s", curveadm.ClusterType())
```

Comment:
How display if previous cluster has no type filed.

Reply:
I give it a default value ‘develop’